src/lib/Libcmds/batch_status.c

```diff
@@ -126,3 +126,20 @@ bs_find(struct batch_status *bs, const char *name)
 
 	return bs;
 }
+
+/**
+ * @brief
+ *	init_bstat - Initialize batch status
+ *
+ * @param[in] bstat - batch_status struct
+ *
+ * @return 	void
+ */
+void
+init_bstat(struct batch_status *bstat)
```

Comment:
i think we should follow the name scheme in this file and name this 'bs_init' instead

Reply:
Renamed this as bs_init.